### ISSUE TITLE
Continuous Integration

### ISSUE THREAD

**Issue by theopak** [theopak/steno#1]
Travis-CI, better hosting, distributed backend.

**Comment by derek-schultz** [comment]
Distributed backend?

**Comment by theopak** [comment]
Yeah, my original description was pretty vague. I don't know what I was thinking when I wrote "distributed backend". I updated it to specify that we need the backend services to parse new podcasts as they become available, and then insert that information into the data storage/search without downtime.
